AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/vo/DeviceSetup.kt

```diff
@@ -12,7 +12,7 @@ data class DeviceSetup(
     /**
      * Additional APKs to install before running test.
      */
-    val additionalApks: Set<UploadedApk>? = null,
+    val additionalApks: Set<CloudApk>? = null,
```

Comment:
if order matters, this should be a list instead of a set

Reply:
done